gettingstarted.html

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title: Getting Started
+permalink: /gettingstarted/index.html
+---
+<h1>GETTING STARTED</h1>
+
+
+<h1>Using the Frame</h1>
```

Comment:
Please try and only have 1 h1, then your sub headings are h2, sub sub headings are h3, etc.